### PR TITLE
Update HipChat integration coverage

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/410930961418306

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://40-bb-docs.surge.sh

File: integrations/hipchat.adoc [hunk 1]
```diff
@@ -1,53 +1,558 @@
...
+page.
 
-Click on **Build your own integration.**
+. Log in to the link:https://dashboard.buddybuild.com/[buddybuild
+  dashboard].
 
-image:img/hipchat-3.jpg[,1500,713]
+. image:../builds/img/button-app_settings.png["The App Settings button",
+  118, 64, role="right"]
+  Click **App Settings** button in the top toolbar. The **Default build
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: Click "the" App Settings button

File: integrations/hipchat.adoc [hunk 2]
```diff
@@ -1,53 +1,558 @@
...
+== Edit notification configuration
+
+Individual notification settings can be adjusted at any time.
+Adjustments take effect immediately for all future notifications.
+
+
+=== All branches
+
+You can only adjust individual settings in the **all branches** panel,
+or that <<remove_config>>; there are no other editing options.
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: Same comments as in Slack, regarding "remove" vs "delete". 

You can only adjust individual settings in the all branches panel, or delete a notification configuration; there are no other editing options.

File: integrations/hipchat.adoc [hunk 3]
```diff
@@ -1,53 +1,558 @@
...
+
+
+=== Branch-specific notifications
+
+Once branch-specific notifications settings are established, it is not
+possible to adjust which branch they apply to. If you need to accomplish
+this, create branch-specific settings for the "new" branch and then
+remove the original branch-specific settings. See <<per-branch>> and
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 4]
```diff
@@ -1,53 +1,558 @@
...
+<<remove_config>> for details.
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 5]
```diff
@@ -1,53 +1,558 @@
...
+image:img/button-save.png["The Save button", 51, 40, role="right"]
+Adjust the pattern as necessary (see <<pattern>> for more details), then
+click the **Save** button.
+
+
+[[remove_config]]
+== Remove notification configuration
+
+image:img/button-trashcan.png["The Trashcan icon", 57, 47, role="right"]
+When you need to remove notification configuration settings,
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 6]
```diff
@@ -1,53 +1,558 @@
...
+image:img/screen-edit_branch_pattern.png["The Edit branch pattern
+screen", 1280, 354, role="frame"]
+
+image:img/button-save.png["The Save button", 51, 40, role="right"]
+Adjust the pattern as necessary (see <<pattern>> for more details), then
+click the **Save** button.
+
+
+[[remove_config]]
+== Remove notification configuration
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 7]
```diff
@@ -1,53 +1,558 @@
...
+image:img/button-trashcan.png["The Trashcan icon", 57, 47, role="right"]
+When you need to remove notification configuration settings,
+for all branches, specific branches, or pattern-specific branches or
+tags, click the **Trashcan** icon at the top right of the appropriate
+panel. The panel is removed, and notifications based on the removed
+configuration cease immediately.
+
+
+[[remove_connection]]
+== Remove room connection
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 8]
```diff
@@ -1,53 +1,558 @@
...
+
+The steps to remove a room connection differ between connections that
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 9]
```diff
@@ -1,53 +1,558 @@
...
+
+
+[[remove_config]]
+== Remove notification configuration
+
+image:img/button-trashcan.png["The Trashcan icon", 57, 47, role="right"]
+When you need to remove notification configuration settings,
+for all branches, specific branches, or pattern-specific branches or
+tags, click the **Trashcan** icon at the top right of the appropriate
+panel. The panel is removed, and notifications based on the removed
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete", "deleted"

File: integrations/hipchat.adoc [hunk 10]
```diff
@@ -1,53 +1,558 @@
...
+. image:img/button-hipchat.png["The HipChat button", 210, 32, role="right"]
+  In the left navigation, click the **HipChat** button. The **HipChat**
+  screen is displayed, showing all of the shared HipChat rooms:
++
+image:img/screen-hipchat.png["The HipChat screen", 1280, 569,
+role="frame"]
+
+. image:img/button-trashcan.png["The Trashcan icon", 57, 47, role="right"]
+  Click the **Trashcan** icon beside the room connection you wish to
+  remove. A delete confirmation dialog appears:
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

File: integrations/hipchat.adoc [hunk 11]
```diff
@@ -1,53 +1,558 @@
...
+. <<navigate>>
+
+. Click the **Connected rooms** tab. The connected room are displayed:
++
+image:img/panel-hipchat-connected_rooms.png["The list of connected rooms",
+905, 219]
+
+. image:img/button-trashcan.png["The Trashcan icon", 57, 47, role="right"]
+  Click the **Trashcan** icon beside the room connection you wish to
+  remove. A delete confirmation appears:
```
Comment (benedict-jw) [inline integrations/hipchat.adoc]: "delete"

**Comment by bbdocs** [comment]
See the website here: http://40-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://40-bb-docs.surge.sh

**Pull request merged**